node_modules/oae-authz/lib/test/util.js

```diff
@@ -16,6 +16,114 @@
 var _ = require('underscore');
 var assert = require('assert');
 
+var AuthzAPI = require('oae-authz');
+var AuthzDelete = require('oae-authz/lib/delete');
+
+/**
+ * Delete the given resource by its id and ensure it completes successfully
+ *
+ * @param  {String}         resourceId      The id of the resource to try and delete
+ * @param  {Function}       callback        Invoked when all assertions pass
```

Comment:
For the `assert...` methods, that tag line hasn't been used because it's not our standard callback function that contains an error as the first parameter.